Core/include/Acts/TrackFinding/MeasurementSelector.ipp

```diff
@@ -28,24 +28,17 @@ MeasurementSelector::select(
 
   // Get geoID of this surface
   auto geoID = candidates.front().referenceSurface().geometryId();
+  // Get the eta of the first track state
+  const auto eta =
+      std::atanh(std::cos(candidates.front().predicted()[eBoundTheta]));
```

Comment:
Is there no utility function for eta ?

Reply:
I fear only for vectors. I can check again

Reply:
looks like there is no such function